src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -659,7 +659,35 @@ paths:
         connectionType: VPC_LINK
         connectionId: '${vpc_connection_id}'
         httpMethod: POST
-
+    patch:
+      summary: Updates specific details within an Anonymous Elector Document (AED).
+      description: Updates specific details (e.g. the elector's phone number) within an Anonymous Elector Document (AED). Note that if more than one AED has been issued for an application, then all of them will be updated.
+      tags:
+        - Anonymous Elector Documents (AED)
+      requestBody:
+        $ref: '#/components/requestBodies/UpdateAnonymousElectorDocument'
+      responses:
+        '204':
+          description: AED updated successfully
```

Comment:
Minor: All AEDs with given `sourceReference` updated successfully as it could update more than 1 AEDs

Reply:
I've just gone for "Updated successfully" :)

I know I briefly mentioned that more than one AED may be updated, but it feels like an implementation detail. Aside from anything else, I assume the vast majority of applications will only have one AED.